packages/alias/test/test.js

```diff
@@ -348,3 +265,39 @@ test('Local customResolver plugin-like object', (t) => {
 
   t.is(resolved, localCustomResult);
 });
+
+
+test("Works in rollup with external resolving plugin", t =>
+  rollup({
+    input: "./test/fixtures/index.js",
+    plugins: [
+      alias({
+        entries: [
+          { find: "fancyNumber", replacement: "./aliasMe" },
+          { find: "./anotherFancyNumber", replacement: "./localAliasMe" },
+          { find: "numberFolder", replacement: "./folder" },
+          { find: "./numberFolder", replacement: "./folder" }
+        ]
+      }),
+      {
+        name: "external-resolving-plugin",
+        resolveId: () => "./test/fixtures/folder/anotherNumber.js"
+      }
+    ]
+  })
+    .then(getModuleIdsFromBundle)
```

Comment:
Maybe I just don't understand what exactly is being tested here and why this is needed but this test seems to be overly complex for what it should test (which is "Works in rollup with external resolving plugin"). Couldn't this be simplified?

Reply:
`this.resolve()` is tested here. As mentioned in previous discussion, Rollup has to go through other plugins, and this behaviour is tested here. So this test covers correct integration with Rollup's API.

Reply:
I assume what it tests here - but still, this test seems a little bit convoluted for me. From what I understand the resolving plugin here just forces a particular file for everything. 

Personally I would just prepare a separate fixture for this test rather than reusing the existing one, because it only confuses me right now. But even if we keep reusing it then lines testing what `moduleIds` here is are rather complicated for what it is. It's a single ID if but yet there is mapping, sorting, mapping+forEaching.

Reply:
i will rework this completely according to other comments, so closing it